.gitignore

```diff
@@ -84,6 +84,10 @@ venv/
 ENV/
 
 # config files
+client_secret_*
 fb_client_secrets.json
 google_client_secrets.json
-monster_config.py
\ No newline at end of file
+monster_config.py
```

Comment:
Minor point: the changes in this file are unrelated to monster edit functionality.  It's preferred to keep each PR to just one function.  No big deal this time, but worth keeping in mind for next time.